{{cookiecutter.project_slug}}/requirements/dev-environment.yml

```diff
@@ -0,0 +1,138 @@
+# This file may be used to create an environment using:
+# $ conda create --name <env> --file <this file>
+# platform: linux-64
+channels:
```

Comment:
If not already done (*), mention in docs that channels and their order specified in `*.yml` files are not remembered by an environment after its creation, but only respected during `conda env create -f ...`.

_(*) I'll go through the docs later._